cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunner.java

```diff
@@ -133,7 +131,9 @@ public ProgramController run(final Program program, ProgramOptions options) {
                                 : System.currentTimeMillis();
 
     String workflowBatch = arguments.getOption(ProgramOptionConstants.WORKFLOW_BATCH);
-    String adapterName = arguments.getOption(ProgramOptionConstants.ADAPTER_NAME);
+    final String adapterName = arguments.getOption(ProgramOptionConstants.ADAPTER_NAME);
+    final String twillRunId = arguments.getOption(ProgramOptionConstants.TWILL_RUN_ID);
```

Comment:
The `twillRunId` is only used for passing it to store, which accepts in the string format, hence using `String` here.